config/locales/en.yml

```diff
@@ -1,33 +1,34 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t "hello"
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t("hello") %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   "true": "foo"
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  service:
+    name: Review criminal legal aid applications
+
+  views:
+    pagination:
+      title: "Pagination navigation"
+      first_html: "<span class='govuk-pagination__link-title'>First <span class='govuk-visually-hidden'>page</span></span>"
+      last_html: "<span class='govuk-pagination__link-title'>Last <span class='govuk-visually-hidden'>page</span></span>"
+      previous_html: "<svg class='govuk-pagination__icon govuk-pagination__icon--prev' xmlns='http://www.w3.org/2000/svg' height='13' width='15' aria-hidden='true' focusable='false' viewBox='0 0 15 13'><path d='m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z'></path></svg><span class='govuk-pagination__link-title'>Previous <span class='govuk-visually-hidden'>page</span></span>"
+      next_html: "<span class='govuk-pagination__link-title'>Next <span class='govuk-visually-hidden'>page</span></span><svg class='govuk-pagination__icon govuk-pagination__icon--next' xmlns='http://www.w3.org/2000/svg' height='13' width='15' aria-hidden='true' focusable='false' viewBox='0 0 15 13'><path d='m8.107-0.0078125-1.4136 1.414 4.2926 4.293h-12.986v2h12.896l-4.1855 3.9766 1.377 1.4492 6.7441-6.4062-6.7246-6.7266z'></path></svg>"
```

Comment:
Any reason we are storing html in the locales file?

Reply:
no idea, I copied and pasted this from provider that come from crime apply.. not even 100% sure we use these ones... :( 

let me check and get back to you

Reply:
removed - though will likely come back in later when we do pagination but worth a deeper review of the approach at that stage.